HIRS_ProvisionerTPM2/package/postinst

```diff
@@ -1,6 +1,7 @@
 HIRS_SITE_CONFIG="/etc/hirs/hirs-site.config"
 
 mkdir -p /var/log/hirs/provisioner
+mkdir -p /etc/hirs/provisioner/certs
 ln -s -f /usr/local/bin/hirs-provisioner-tpm2 /usr/sbin/hirs-provisioner-tpm2
 ln -s -f /usr/local/bin/tpm_aca_provision /usr/sbin/tpm_aca_provision
 
```

Comment:
Wouldn't you need the `ln -s -f /usr/local/bin/tpm_version /usr/sbin/tpm_version` for this Ubuntu package as well?

Reply:
I didn't have a definitive answer on this, so I just went ahead and added it.